base/changes.txt

```diff
@@ -6,12 +6,19 @@ to completeness or accuracy and it contains some references to files that are
 not part of the distribution.
 ================================================================================
 
+2024-06-21  Yukai Chou  <muzimuzhi@gmail.com>
+
+	* ltpara.dtx
+	Append \everypar toks to \g__parar_standard_everypar_tl, rollback
+	2023/06/01 (gh/1386)
+
 2024-06-19  David Carlisle  <David.Carlisle@latex-project.org>
 
 	* fontdef.dtx:
 	preload ts1lmr.fd in Unicode engines
 
-2024-06-19 Joseph Wright  <Joseph.Wright@latex-project.org>
+2024-06-19  Joseph Wright  <Joseph.Wright@latex-project.org>
```

Comment:
Why the unrelated change here?

Reply:
To unify whitespaces. I can drop this unrelated change.